homeassistant/components/light/mqtt.py

```diff
@@ -44,14 +45,18 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
             'on': config.get('payload_on', DEFAULT_PAYLOAD_ON),
             'off': config.get('payload_off', DEFAULT_PAYLOAD_OFF)
         },
-        config.get('optimistic', DEFAULT_OPTIMISTIC))])
+        config.get('optimistic', DEFAULT_OPTIMISTIC),
+        {key: float(config.get(key + '_scale', DEFAULT_SCALE))
```

Comment:
Ah, fair enough. I can easily remove that.
On Fri, Feb 26, 2016 at 08:22 Paulus Schoutsen <notifications@github.com>
wrote:

> In homeassistant/components/light/mqtt.py
> <https://github.com/balloob/home-assistant/pull/1403#discussion_r54264601>
> :
>
> > @@ -44,14 +45,18 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
> >              'on': config.get('payload_on', DEFAULT_PAYLOAD_ON),
> >              'off': config.get('payload_off', DEFAULT_PAYLOAD_OFF)
> >          },
> > -        config.get('optimistic', DEFAULT_OPTIMISTIC))])
> > +        config.get('optimistic', DEFAULT_OPTIMISTIC),
> > +        {key: float(config.get(key + '_scale', DEFAULT_SCALE))
>
> I can understand scale for brightness but rgb ? That's pretty much
> standardized to 255.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/1403/files#r54264601>.
>
-- 
-St. John